src/tucal/plugins/eventHTU.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import json
+
+import tucal
+import tucal.db
+import tuwien.sso
+
+query = {
+  "operationName": "FetchEvents",
+  "variables": {
+    "page": 1,
+    "limit": 50
+  },
+  "query": "query FetchEvents($orderBy: EventOrderBy, $direction: SortDirection, $page: Int, $limit: Int) { events(orderBy: $orderBy, direction: $direction, page: $page, limit: $limit) { total elements { id url title description beginsOn endsOn status picture { id url } physicalAddress { id description locality } tags { ...TagFragment } } }} fragment TagFragment on Tag { id title}"
+}
+
+HTU_HOST = 'events.htu.at'
+HTU = f'https://{HTU_HOST}'
+
+
+class EVENTS(tucal.Plugin):
+    @staticmethod
+    def sync():
+        url = f'{HTU}/api'
+        r = requests.post(url, json=query)
+        if r.status_code != 200:
+            raise RuntimeError()
+
+        cur = tucal.db.cursor()
+
+        raw_events = r.json()
+        events = raw_events['data']['events']['elements']
+
+        for event in events:
+            data = {
+                'id': f'htu-{event["id"]}',
+                'start': event['beginsOn'],
+                'end': event['endsOn'],
+                'room': event['url'],
+                'del': not (event['status'] == 'CONFIRMED'),
+                'data': json.dumps(event)
```

Comment:
* `room` sould only be a room_nr. This will never be the case in this plugin, so set it `NULL` in the query below.
* `id` may be `event['id']` - the primary key of the table is `(source, event_id)`
* `del` should only be used to mark a event as deleted. Set it to False or omit it below in the query
* `data` should be structured like `{"htu": {...}}` to avoid overwritting from other sources
* `start` and `end` should be `datetime.datetime` objects. So please parse your timestamps (and make sure, they have the right timezone set!)

Please also consider to update `event_merger.py`'s `update_event()` to add a `summary` field :)